python/smqtk/utils/configuration.py

```diff
@@ -60,10 +60,26 @@ def get_default_config(cls):
         turning those argument names into configuration dictionary keys. If any
         of those arguments have defaults, we will add those values into the
         configuration dictionary appropriately. The dictionary returned should
-        only contain JSON compliant value types.
+        only contain JSON compliant value types. If the default arguments are
+        not json compliant then the function should be overriden to convert
+        the convert the default to a json-compliant stand in.
 
         It is not be guaranteed that the configuration dictionary returned
         from this method is valid for construction of an instance of this class.
+        >>> class NonCompliantDefault(Configurable):
```

Comment:
The doctests are failing due to math.pow not being defined within the doctest context. You'll have to add an import math above your class definition here.

```suggestion
        >>> import math
        >>> class NonCompliantDefault(Configurable):
```